bundles/org.openhab.binding.speedtest/src/main/java/org/openhab/binding/speedtest/internal/speedtestBindingConstants.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.speedtest.internal;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link speedtestBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Brian Homeyer - Initial contribution
+ */
+@NonNullByDefault
+public class speedtestBindingConstants {
```

Comment:
```suggestion
public class SpeedtestBindingConstants {
```
Should start with a capital. See https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Reply:
done